server/src/routers/documentRoutes.ts

```diff
@@ -245,11 +243,11 @@ class DocumentRoutes {
       this.authenticator.isAdminOrUrbanPlanner,
       body('doc1').isNumeric(),
       body('doc2').isNumeric(),
-      body('link_type').isString(),
+      body('links').isArray({ min: 1 }),
       this.errorHandler.validateRequest,
       (req: any, res: any, next: any) =>
         this.controller
-          .addLink(req.body.doc1, req.body.doc2, req.body.link_type)
```

Comment:
Adesso quindi invi direttamente un array di link types??

Reply:
Yes, the controller receives an array of links of type LinkClient